databases/databases/project_sql_zoo.md

```diff
@@ -5,7 +5,8 @@ SQL Zoo is one of the few resources online that actually lets you build and run
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-  1. Go to [SQL Zoo](https://sqlzoo.net/) and do Tutorials 0-9 listed under the "Tutorial Section" and the quizzes listed at the end of each.  The first is the ["Select" tutorial](https://sqlzoo.net/wiki/SELECT_basics).  Make sure the dropdown on the upper right of the main page for "Engine" says "MySQL" (the default).  Large results will be cut off and not all rows or columns shown, so the "answers" may not look 100% correct.
+  1. Go to [SQL Zoo](https://sqlzoo.net/) and do Tutorials 0-9 listed under the "Tutorial Section" and the quizzes listed at the end of each.  The first is the ["Select" tutorial](https://sqlzoo.net/wiki/SELECT_basics).  Make sure the dropdown on the upper right of the main page for "Engine" says "MySQL" (the default).  Large results will be cut off and not all rows or columns shown, so the "answers" may not look 100% correct.s
```

Comment:
Looks like there is an additional "s" at the end of this sentence that is not needed.

Reply:
fixed!